CmakeLists.txt

```diff
@@ -1,11 +1,19 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.8)
 project(cpp_uno)
-
+set (CMAKE_CXX_STANDARD 14)
 set(include_dirs ${include_dirs}
     "${CMAKE_SOURCE_DIR}/include")
 set(sources ${sources}
     "${CMAKE_SOURCE_DIR}/src/main.cpp"
-    "${CMAKE_SOURCE_DIR}/include/main.h")
-
+    "${CMAKE_SOURCE_DIR}/include/main.h"
```

Comment:
seria legal garantir que todos os arquivos da pasta include e da pasta src sejam inclusos, ao invés e adicionar um por um (se tiver um novo arquivo, vai ter que adicionar la de novo)

Reply:
GLOB, o mecanismo de juntar os arquivos sem listar um a um, é fonte de problemas: https://stackoverflow.com/questions/1027247/is-it-better-to-specify-source-files-with-glob-or-each-file-individually-in-cmak/1060061#1060061